packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.mjs

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable @jessie.js/safe-await-separator */
+import { promises as fs } from 'fs';
+import assert from 'assert';
+
+import {
+  waitForBlock,
+  provisionSmartWallet,
+  getUser,
+  executeOffer,
+} from './upgradeHelpers.mjs';
+import {
+  agd,
+  agoric,
+  agops,
+  agopsLocation,
+  executeCommand,
+} from '../cliHelper.mjs';
+import {
+  HOME,
+  ATOM_DENOM,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+} from '../constants.mjs';
+
+const acceptEC = async () => {
+  console.log('ACTIONS Tickling the wallets so they are revived');
+
+  const govAccounts = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+  let voter = 0;
+  for (const account of govAccounts) {
+    const runs = [1, 2];
+    for (const run of runs) {
+      console.log(`${account}: ${run}: Accepting EC Committee`);
+
+      if (run === 1) {
+        try {
+          const params = ['ec', 'committee', '--send-from', account];
+          await executeCommand(agopsLocation, params, {
```

Comment:
This seems to be doing almost the same thing as line 47.  Why is the syntax so different?

Was there a missed opportunity to turn line 39 into the `agops.ec()` form, or is this intended to demonstrate the two styles of call?